_remake/main.js

```diff
@@ -114,27 +123,75 @@ app.use(expressSession({
   }
 }));
 
-
 // flash message middleware
 app.use(flash());
 
+// if app is multi tenant
+// open MySQL connection
+// and use bodyParser (needed for post requests in lib/init-service-routes.js)
+if (RemakeStore.isMultiTenant()) {
+  global.config = {
+    db: {
+      name: "remake-service",
+      port: 3306,
+      host: "localhost",
+      user: "remake",
+      password: "ekamer"
+    },
+    jwt: {
+      // JWT_SECRET is generated by CLI and
+      // exported as an env var by setEnvironmentVariables();
+      // Each multi tenant instance should have a unique JWT secret
+      secret: process.env.JWT_SECRET, 
+      duration: 365 * 24 * 3600 // 1 year
+    },
+    location: {
+      remake: "/opt/remake/remake-deployment",
+      tmp: "/tmp/remake"
+    },
+    limits: {
+      appPerUser: 10,
+    }
+  }
+  
+  // open MySQL connection
+  app.use((req, res, next) => {
+    global.connection = mysql.createConnection({
+      host : config.db.host,
+      user : config.db.user,
+      password : config.db.password,
+      database : config.db.name,
+      port: config.db.port
+    });
+  
+    connection.connect((err) => {
+      if (err) throw err;
+      else next();
+    });
+  });
+  app.use(bodyParser.json());
+  app.use(bodyParser.urlencoded({ extended: false }));
```

Comment:
We don't need these two lines (`172` and `173`). We've already included them on lines `31` and `32` as:

```
app.use(express.urlencoded({ extended: false }));
app.use(express.json());
```

Reply:
👍

Reply:
Added at merge 😄